src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -316,9 +316,9 @@ function SubmissionsFactory(
                 experimentalCsvGenerator.addRecord(csvRecord.submissionData)
```

Comment:
making a note that `csvRecord.submissionData` is an internal property - how about writing a getter method instead?